contents/blog/product-engineer-tools.md

```diff
@@ -0,0 +1,90 @@
+---
+date: 2022-12-14
+title: The best tools for product engineers
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+categories: ["Startups", "Guides"]
+---
+
+Building tools for product engineers is a big trend right now. Every week brings a flashy launch of a new tool intersecting product and engineering. This is because building a great product users value is a proven way to succeed as a business. Helping product engineers make this happen is valuable.
```

Comment:
```suggestion
Building tools for [product engineers](/blog/what-is-a-product-engineer/) is a big trend right now. Every week brings a flashy launch of a new tool intersecting product and engineering. This is because building a great product users value is a proven way to succeed as a business. Helping product engineers make this happen is valuable.
```

Reply:
For SEO reasons, always link on the earlier reference in an article, even if you have a link in the footer.